credential_helper.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export GIT_USER="enter your git username here"
+export GIT_USER="wendywwang"
```

Comment:
Shouldn't get changed

Reply:
@drewmibm ?! I didn't intend to include this file.. let me fix it

Reply:
done